client/engine/chainservice/simulated_backend_chainservice_test.go

```diff
@@ -1,16 +1,75 @@
 package chainservice
 
 import (
+	"bytes"
 	"math/big"
 	"testing"
 
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/google/go-cmp/cmp"
+	"github.com/statechannels/go-nitro/channel/state"
+	"github.com/statechannels/go-nitro/channel/state/outcome"
 	"github.com/statechannels/go-nitro/protocols"
 	"github.com/statechannels/go-nitro/types"
 )
 
-func TestSimulatedBackendChainService(t *testing.T) {
+type actor struct {
+	Address    types.Address
+	PrivateKey []byte
+}
+
+// actors namespaces the actors exported for test consumption
+type actors struct {
+	Alice actor
+	Bob   actor
+}
+
+// Actors is the endpoint for tests to consume constructed statechannel
+// network participants (public-key secret-key pairs)
+var Actors actors = actors{
```

Comment:
I think we have some of this boilerplate in the codebase already, which we could reuse here.

Reply:
Much obliged!